runtime/compiler/env/VMJ9.cpp

```diff
@@ -1361,6 +1361,29 @@ TR_J9VMBase::getReferenceElement(uintptrj_t objectPointer, intptrj_t elementInde
    return (uintptrj_t)J9JAVAARRAYOFOBJECT_LOAD(vmThread(), objectPointer, elementIndex);
    }
 
+int32_t TR_J9VMBase::getArrayType(TR_OpaqueClassBlock* clazz)
+   {
+   J9Class* j9clazz = (J9Class*)clazz;
+   if (j9clazz == jitConfig->javaVM->booleanReflectClass)
+      return 4;
+   else if (j9clazz == jitConfig->javaVM->charReflectClass)
+      return 5;
+   else if (j9clazz == jitConfig->javaVM->floatReflectClass)
+      return 6;
+   else if (j9clazz == jitConfig->javaVM->doubleReflectClass)
+      return 7;
+   else if (j9clazz == jitConfig->javaVM->byteReflectClass)
+      return 8;
+   else if (j9clazz == jitConfig->javaVM->shortReflectClass)
+      return 9;
+   else if (j9clazz == jitConfig->javaVM->intReflectClass)
+      return 10;
+   else if (j9clazz == jitConfig->javaVM->longReflectClass)
+      return 11;
```

Comment:
Is this set of magic numbers common practice in the JIT?  Is there a named constant / enum that could be used instead?